client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html

```diff
@@ -1,207 +1,198 @@
-                            data-ng-minlength="ctrl.constants.passwordMinLength" data-ng-model="ctrl.migrate.password">
-                            <small class="help-block" data-i18n-static="email_tab_modal_create_account_account_password_error" data-ng-show="ctrl.migrateDestinationForm.accountPassword.$dirty && ctrl.migrateDestinationForm.accountPassword.$invalid"></small>
-                            
-                        </div>
-
-                        <!-- Aknowledge Checkbox -->
-                        <div class="oui-checkbox">
-                            <input 
-                                type="checkbox" 
-                                id="aknowledgeCheckbox" 
-                                class="oui-checkbox__input" 
-                                data-ng-model="ctrl.migrate.aknowledge"
-                            >
-                            <label class="oui-checkbox__label-container" for="aknowledgeCheckbox">
-                                <span class="oui-checkbox__label">
-                                    <span data-ng-bind="ctrl.translator.tr('email_tab_modal_migrate_aknowledge')"></span>
-                                    <span class="oui-checkbox__icon">
-                                        <i class="oui-icon oui-icon-checkbox-unchecked" aria-hidden="true"></i>
-                                        <i class="oui-icon oui-icon-checkbox-checked" aria-hidden="true"></i>
-                                        <i class="oui-icon oui-icon-checkbox-checkmark" aria-hidden="true"></i>
-                                        <i class="oui-icon oui-icon-checkbox-indeterminate" aria-hidden="true"></i>
-                                    </span>
-                                </span>
-                            </label>
-                        </div>
-                        
+            <!-- No Service -->
+            <div data-ng-if="!ctrl.availableServices">
+                <p data-i18n-static="email_tab_modal_migrate_no_email_service"></p>
+                <p>
+                    <a href="{{::ctrl.emailsOrder}}" target="_blank" class="btn btn-primary">
+                        <span data-i18n-static="email_tab_modal_migrate_order_email_service"></span>
+                        <span class="fa fa-external-link"></span>
+                    </a>
+                </p>
+            </div>
+            <!-- /No Service -->
+
+            <!-- Has Services -->
+            <div class="oui-field"
+                data-ng-if="ctrl.availableServices">
+                <label class="oui-field__label oui-label"
+                    data-i18n-static="email_tab_modal_migrate_destination_service_label">
+                </label>
+                <div class="row oui-field__control oui-field__control_auto">
+                    <div class="col-md-{{(12 / ctrl.availableServices)}}"
+                        data-ng-repeat="(serviceType, destinationService) in ctrl.destinationServices"
```

Comment:
Could you add a `track by` on your `data-ng-repeat`?

Reply:
Done